other/glad/include/vulkan/vulkan.h

```diff
@@ -0,0 +1,4318 @@
+/**
```

Comment:
glad/vulkan.h and vulkan/vulkan.h.. confusing

Reply:
They're identical... just needed to satisfy how different cpp files include this header, and this was the simplest way.

Reply:
maybe it'd be better to copy it as part of the cmake hunter build (so there's only one copy in the repo)